Assets/Scripts/Mechanics/HitboxScript.cs

```diff
@@ -6,6 +6,7 @@
 public class HitboxScript : MonoBehaviour
 {
     public bool active = true;
+    public bool destroyOnHit = false;
```

Comment:
Is this used?

Reply:
Yes, it's used by the basic projectile. Basically just to make sure it'll disappear once it hits something.